### ISSUE TITLE
Class overview contains {@template xxx} of properties/methods

### ISSUE THREAD

**Issue by goderbauer** [dart-lang/dartdoc#2272]
When the doc comment of a property or method within a class starts with `{@template fff}`, that string appears in the overview doc of the enclosing class:

https://master-api.flutter.dev/flutter/widgets/SliverFixedExtentList-class.html

![image](https://user-images.githubusercontent.com/1227763/88341911-4c73d400-ccf3-11ea-83ca-74133ca7616f.png)

**Comment by srawlins** [comment]
Interesting that it doesn't show up in the property's individual page.

Can you link me to the source code of SliverFixedExtentList?

**Comment by goderbauer** [comment]
Here's the source: https://github.com/flutter/flutter/blob/b58c3f0852863417b482147527e8b694243f3b2f/packages/flutter/lib/src/widgets/sliver.dart#L871

**Comment by goderbauer** [comment]
There're about 15 occurrences of this in the Flutter docs. It also happens with library overview pages: https://master-api.flutter.dev/flutter/cupertino/cupertino-library.html (search for `@template`).

**Comment by srawlins** [comment]
Do you expect the documentation to be the text inside `{@template} ... {@endtemplate}`, as if it were rendered with `@macro`?

**Comment by goderbauer** [comment]
> Do you expect the documentation to be the text inside `{@template} ... {@endtemplate}`

Yes, I would expect the class/library overview to show an excerpt of what's inside the template directive. For the SliverFixedExtend example from above, I would expect it to just show: "The delegate that provides the children for this widget."

**Comment by srawlins** [comment]
Excellent. Thanks.

**Comment by jcollins-g** [comment]
This is a tricky one.  This case _appears_ to be well tested in dartdoc and behaves appropriately in similarly constructed test scenarios.  So there's something special about these handful of cases that is failing here.

**Comment by srawlins** [comment]
Yes! I too failed to reproduce with a unit test. Something about inherited methods, inheritance between libraries... inheritance across the public/private boundary? Indirect inheritance?

**Comment by jcollins-g** [comment]
@srawlins now that I have the debugger working I'll dig into this more tomorrow with the actual flutter code and see if I can step through it in debug mode to see what's happening.
